examples/hybrid/cli_options.jl

```diff
@@ -24,6 +24,20 @@ function parse_commandline()
         help = "Moisture model [`dry` (default), `equil`, `non_equil`]"
         arg_type = String
         default = "dry"
+        "--micro"
+        help = "Microphysics model [`nothing` (default), `0M`]"
+        arg_type = String
+        "--HSF"
+        help = "Held Suarez Forcing [`false` (default), `true`]"
+        arg_type = Bool
+        default = false
+        "--turbconv"
+        help = "Turbulence Convection model [`nothing` (default), `const_diffusivity`]"
+        arg_type = String
+        "--idealized_h2o"
```

Comment:
Can we make this an option in `--rad`

Reply:
Sounds good to me, how can we do that?

Reply:
I'm not sure how to do this off the top of my head, so I'm going to save this for a separate PR.

Reply:
might need some inputs from zhaoyi and dennis